extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -706,6 +706,38 @@ public function execute() {
 			wfProfileOut( __METHOD__."-varsdb" );
 		}
 
+		#Fix shared uploads to UCP wikis
+		if (
+			!empty( $this->mVariables['wgUseSharedUploads'] ) &&
+			!empty( $this->mVariables['wgSharedUploadDBname'] ) &&
+			$this->mWikiID !== 177
+		) {
+			$partnerWikiData = $dbr->selectRow(
+				array( "city_list" ),
+				array(
+					"city_id",
+					"city_url",
+					"city_dbname",
+					"city_path",
+				),
+				array( "city_dbname" => $this->mVariables[ 'wgSharedUploadDBname' ] )
```

Comment:
These lines within the `selectRow` call should all use short array syntax `[]` and single quotes per coding conventions.